app/controllers/comment_controller.py

```diff
@@ -63,3 +64,12 @@ def comments(id):
         post=post,
         comments=comments,
     )
+
+
+@comment.route("/comment/<int:post_id>/<int:id>", methods=["GET"])
+@requires_roles("admin", "momod")
+def mark_spam(post_id, id):
+    comments = Comment.query.get(int(id))
```

Comment:
oiya ini cuma satu, harusnya `comment` aja instead of `comments`

Reply:
oh takut overwrite blueprint nya ya...

Reply:
yep